app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Typography, Dropdown, Tag, Tooltip, Avatar } from "neetoui";
+
+const Note = ({
+  id,
+  title,
+  description,
+  tag,
+  status,
+  created_at,
```

Comment:
Why have you used snake_case for props? I saw similar naming in multiple files.

Reply:
This is resolved.

I was trying to retain structure of note object which is why used object properties as props.